project/AppDependencies.scala

```diff
@@ -22,10 +22,10 @@ object AppDependencies {
 
   val compile: Seq[ModuleID] = Seq(
     ws exclude("org.apache.httpcomponents", "httpclient") exclude("org.apache.httpcomponents", "httpcore"),
-    "uk.gov.hmrc"       %% "bootstrap-backend-play-28" % "5.7.0",
-    "org.typelevel"     %% "cats-core"         % "2.6.1",
-    "com.chuusai"       %% "shapeless"         % "2.4.0-M1",
-    "com.typesafe.play" %% "play-json-joda"    % "2.9.2"
+    "uk.gov.hmrc"       %% "bootstrap-backend-play-28" % "5.20.0",
+    "org.typelevel"     %% "cats-core"                 % "2.6.1",
+    "com.chuusai"       %% "shapeless"                 % "2.4.0-M1",
+    "com.typesafe.play" %% "play-json-joda"            % "2.9.2"
```

Comment:
cats-core can go to 2.7.0
flexmark-all can go to 0.62.2
scalamock can go to 5.2.0
wiremock-jre8 can go to 2.32.0